deployment/helm/templates/opcua-configuration.yaml

```diff
@@ -57,8 +78,11 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 6 }}
       {{- end }}
-      {{- if .Values.opcua.configuration.mountCertificates }}
+      {{- with .Values.opcua.configuration.brokerPod.volumes}}
```

Comment:
is this a breaking change?  should the version update reflect that?

Reply:
Good point. just modified it in 5d493dcd0f006713 to make it not breaking so you can specify `mountCertificates` for OPC and/or other volumes